### ISSUE TITLE
3.3.1 Error identification

### ISSUE THREAD

**Issue by svetlanaangelova** [samheather/iapt2group1#102]
- Errors should be displayed at the top of the page as well as next to each field (affects Register, Login, Create project, Populate project, Add field, Add image). [Source](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G85)

**Comment by svetlanaangelova** [comment]
Add a one-liner "There are some errors with your submission. Please see individual fields for more information."
